packages/material-ui/src/MenuList/MenuList.test.js

```diff
@@ -28,7 +28,6 @@ describe('<MenuList />', () => {
     });
 
     it('should render a List', () => {
-      assert.strictEqual(wrapper.name(), 'List');
```

Comment:
Not sure why you would test here for that specific component. Would `ul` be enough or some html element that matches those semantics?